WeakAuras/Types.lua

```diff
@@ -3151,6 +3151,28 @@ LSM:Register("statusbar", "Thick Stripes", [[Interface\AddOns\WeakAuras\Media\Te
 LSM:Register("statusbar", "Thin Stripes", [[Interface\AddOns\WeakAuras\Media\Textures\Statusbar_Stripes_Thin]])
 LSM:Register("border", "Drop Shadow", [[Interface\AddOns\WeakAuras\Media\Textures\Border_DropShadow]])
 
+if PowerBarColor then
+  local function capitalizeFirstLetter(str)
+    -- Split the string into words separated by underscores
+    local words = {}
+    for word in string.gmatch(str, "[^_]+") do
+      table.insert(words, word)
+    end
+    -- Capitalize the first letter of each word
+    for i, word in ipairs(words) do
+      words[i] = word:sub(1, 1):upper() .. word:sub(2):lower()
+    end
+    return table.concat(words, " ")
+  end
+
+  for power, data in pairs(PowerBarColor) do
+    if type(power) == "string" and data.atlas then
```

Comment:
There are apparently more files here which can be deduced from the atlasElementName. There are lots of variants of those files for various settings, bit I think including:
"UI-HUD-UnitFrame-Player-PortraitOff-Bar" .. data.atlasElementName would probably be a good idea. All the other variants aren't imho that useful.